lib/matplotlib/_backports/numpy.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+
+# Copy-pasted from numpy.lib.stride_tricks 1.11.2.
```

Comment:
Can you add this comment on all functions? This will help in the future if any others are added here and these need to be removed.